polyglot-kotlin/src/test/resources/unit-tests/dependencies/variation-8/pom.kts

```diff
@@ -0,0 +1,17 @@
+project {
+    profiles {
+        profile {
+            id("other")
```

Comment:
In order to keep the same style with `plugin("gav")` I suggest to have the same here profile("other"). The same was done to `project("id")` already

Reply:
This style is actually supported, but it looks like it isn't covered in the unit tests. I'll correct that.

Reply:
ok. From my point of view, id as a property looks better comparing to function call